docs/writing/posts/consulting.md

```diff
@@ -0,0 +1,207 @@
+---
+draft: False
+date: 2024-01-22
+slug: consulting
+categories:
+  - Personal
+authors:
+  - jxnl
+---
+
+# What I Learned from Indie Consulting
+
+!!! note "Fragments"
+
+    This is just a note to the reader that I like to write in a fragmented style using speech to text so if you see any errors or don't really understand what's going on feel free to just contribute and edit or leave a question on the github repo.
+
+I specify indie consulting as something that is completely and wholly separate from the big-time consulting we hear about from those ridiculous institutions. Check out this video roasting [McKinsey](https://www.youtube.com/watch?v=AiOUojVd6xQ) From John Oliver to understand how I feel about many of these folks. Theres another great [video](https://www.youtube.com/watch?v=vZE0j_WCRvI) that i saw on tiktok.
+
+<!-- more -->
+
+!!! warning "The failure of big time consulting"
+
+    The difference between Indie Consulting and Big Time Consulting as I define it can be captured in one simple idea. Something that Steve Jobs once talked about:
+
+    > "I don't think there's anything inherently evil in consulting. I think that without owning something over an extended period of time, like a few years, where one has a chance to take responsibility for one's recommendations, where one has to see one's recommendations through all action stages and accumulate scar tissue for the mistakes and pick oneself up off the ground and dust oneself off, one learns a fraction of what one can. Coming in and making recommendations and not owning the results, not owning the implementation, I think is a fraction of the value and a fraction of the opportunity to learn and get better."
+
+I strongly believe that if you're a young person you should not be consulted. You should join a big company or a small company and just eat shit and develop the scar tissue that develops over time.
```

Comment:
```suggestion
I strongly believe that if you're a young person you should not be consulted. You should join a big company or a small company and just eat shit and develop the scar tissue that develops over time.

My friend and fellow indie consultant [Hamel Husain](https://twitter.com/HamelHusain) has another perspective:

> People without meaningful work experience in a particular industry don't add much value (and often negative value) as a consultant.  Large consulting companies like McKinsey, BCG, etc peddle a dangerous form of intellectual masturbation for dysfunctional management and organizations.  Ask yourself, what iconic companies have made impressive pivots or disrupted an industry **because of the advice of McKinsey**? 

>  On the other hand, _good_ consultants can offer specific services and add value in unique ways that employees cannot.  Let me give you some examples:
> - Procurement specialists who will re-negotiate all of your contracts and leverage the fact they are negotiating on behalf of many companies.  I knew someone who did this for large restaurant chains and was able to cut the price of everything from toilet paper to rent by 15% on average.  The suppliers wanted to play nice because this consultant had lots of clients.  Similarly, [Cory Quinn](https://twitter.com/quinnypig) will do this with your cloud bills.
> - In my [consulting practice around LLMs](https://parlance-labs.com/), I often provide vendors with their first customers, broker deals between open source maintainers and commercial platforms, introduce highly qualified candidates that get hired, and other strategic activities that involve working with many different organizations simultaneously.  These activities are in addition to specialized engineering work I do around LLMs.  I believe that the the engineering work alone, while valuable, would NOT justify me as a consultant in the long term.
```